### ISSUE TITLE
mobolienet V2 QAT quantization accuracy is low

### ISSUE THREAD

**Issue by mairkiss** [microsoft/nni#5133]
i use a qat quantizer to quantization my model base on mobolienet v2, the result is always nan at first， so I reduce the learning rate， no more nan. but  the KPI(accuray, recall, precision) of model is low（30% reduction compared to the original model）, how can promote KPI when use QAT quantizer， here is my code：

from learner_curve import Simple_MLSD_Curve_Learner
import json
from nni.algorithms.compression.pytorch.quantization import QAT_Quantizer
from nni.compression.pytorch.quantization.settings import set_quant_scheme_dtype
from nni.compression.pytorch.quantization_speedup import ModelSpeedupTensorRT
from nni.compression.pytorch.utils import count_flops_params
from nni.algorithms.compression.pytorch.quantization import LsqQuantizer
from models.mbv2_mlsd_large_curve import MobileV2_MLSD_Large_Curve
from kpi import calculate_kpi
 configure_list = [{
            'quant_types': ['weight'],
            'quant_bits': {
                'weight': 8,
            },
            'op_types': ['Conv2d']
    }]

    set_quant_scheme_dtype('weight', 'per_channel_symmetric', 'int')
    set_quant_scheme_dtype('output', 'per_tensor_symmetric', 'int')
    set_quant_scheme_dtype('input', 'per_tensor_symmetric', 'int')
    model = get_origin_model(cfg)
    
    dummy_input = torch.randn(1, 12, 272, 480).to(device)
    flops, params, results = count_flops_params(model, dummy_input.to(device))

    optimizer = torch.optim.Adam(params=model.parameters(),lr=cfg.train.learning_rate,weight_decay=cfg.train.weight_decay)
  
    print("model1:", model)
    quantizer = QAT_Quantizer(model, configure_list, optimizer, dummy_input=dummy_input)
    quantizer.compress()

    train(cfg, model, device, optimizer)

    input_shape = (1, 12, 272, 480)
    device = torch.device("cuda")
    
    calibration_config = quantizer.export_model(model_path, calibration_path, onnx_path, input_shape, device) # 
    input_shape = (1, 12, 272, 480)
    calculate_kpi(cfg, model)
    engine = ModelSpeedupTensorRT(model, input_shape, config=calibration_config, batchsize=1,strict_datatype=True, onnx_path="/home/tzc/2021/code/LLD/mlsd_pytorch/qat_model.onnx")  #  
    print('Done')

**Comment by Lijiaoa** [comment]
hi @mairkiss Thanks for reporting this issue! We need some time to run an example to confirm this performance issue `the QAT quantization accuracy is low`. And now we are working on the stable version.
